test/new_relic/agent/instrumentation/rails/action_mailbox_subscriber.rb

```diff
@@ -0,0 +1,123 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require_relative '../../../../test_helper'
+require 'new_relic/agent/instrumentation/action_mailer_subscriber'
+
```

Comment:
Do you know if we're at 100% test coverage for ActionMailbox and ActionMailer?

Reply:
I modeled the testing off of the recent ActiveSupport custom events stuff which delivered 100% coverage, but I haven't checked the coverage for this PR specifically. This gives me an idea to add an optional "branch" scope to Stoat's `coverage` function. Granted, in this case with ActionMailbox we're only see full coverage on the CI run that includes Rails Edge, so that's perhaps a one off scenario that Stoat won't be expected to reach.